ebpfsvc/rpc_api.cpp

```diff
@@ -51,3 +51,55 @@ ebpf_server_verify_and_load_program(
     return EBPF_OPERATION_NOT_SUPPORTED;
 #endif
 }
+
+ebpf_result_t
+ebpf_server_verify_and_authorize_native_image(
+    /* [string][in] */ char* image_path)
```

Comment:
can sal annotation like `_In_z_` be used here?

Reply:
I think it has to match the generated header, which lacks the code analysis annotations.